recipe/meta.yaml

```
@@ -22,22 +22,24 @@ requirements:
   run:
     - python
     - ipykernel
-    - ipywidgets
+    - ipywidgets # [not s390x]
```

It seems like we have the latest on `s390x` and it doesn't seem to depend on anything not on `s390x`. Is there another reason to remove `s390x` support here?
```suggestion
    - ipywidgets
```

---

Ah, well, at first blush jupyter-widgets (name changed from ipywidgets) are used in conjunction with notebooks and lab environments so it isn't needed.  Looking into it further, it seems you can use them outside of that context https://ipywidgets.readthedocs.io/en/7.6.5/embedding.html.  So we could provide it.

---

Though this isn't a normal use case.

---

Any opinions on this @cbouss or @chenghlee ?

---

Ipywidgets is only useful when installed along notebook or jupyterlab. Since both are commented out, this can be commented as well.

---

The other uses seem very specific to me. I would still leave it out.